mqt/bench/utils/tket_helper.py

```diff
@@ -27,8 +27,9 @@ def get_rebase(gate_set_name: str, get_gatenames: bool = False):
         return get_ibm_rebase(get_gatenames)
     elif gate_set_name == "rigetti":
         return get_rigetti_rebase(get_gatenames)
-    #elif gate_set_name == "openqasm":
-        #return get_openqasm_rebase(get_gatenames)
+    # elif gate_set_name == "openqasm":
+    # return get_openqasm_rebase(get_gatenames)
+
```

Comment:
if these lines are no longer needed, they might as well be removed. Also holds for the commented lines below.

Reply:
done